### ISSUE TITLE
400 error when I click on the OAuth Link

### ISSUE THREAD

**Issue by broo0ose** [mapero/node-red-contrib-hangouts#13]
Hi,
I'm trying to set up a new Hangout. I get a 400 error when I click on the OAuth Link. 
Am I missing something? I'm using Chrome logged into my Google account. 
Thanks
Bruce 

Edit: Error: invalid_scope

**Comment by mapero** [comment]
This is a known error due to changes in the authorization process at google. All third party libraries and clients for hangout have this problem. See:

[OAuth invalid scope](https://productforums.google.com/d/msg/hangouts/LU2YZNDhN_s/nF9n8MEXDgAJ)

**Comment by broo0ose** [comment]
thanks for the reply. After a bit of Googling I can see this is a major issue for a number of projects. I guess it kills this node until Google decides to fix it.

**Comment by mapero** [comment]
Thats true for new deployed flows. Existing flows should still work until the token gets invalid.

**Comment by Eeems** [comment]
Looks like there is a workaround according to the link you gave @mapero 
https://github.com/tdryer/hangups/issues/260

**Comment by broo0ose** [comment]
thanks @Eeems that works.

**Comment by juancgalvez** [comment]
Hi @mapero. Do you have any plan to fix it? Did anyone fix it in code that could share?

**Comment by mapero** [comment]
@juancgalvez Hi. I stopped using this and switched to telegram. Google does not provide any API documentation and it is only a matter of time until google changes something again and the hack breaks again. With the workaround it still should work.

Still I am open for pull requests.

**Comment by juancgalvez** [comment]
@mapero. Thank you for the information. I will give it a try and switch.
